src/main/java/duke/DukeStorage.java

```diff
@@ -0,0 +1,127 @@
+package duke;
+
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.FileWriter;
+import java.util.Scanner;
+
+/**
+ * Class that defines the storage for the data needed for Duke
+ *
+ */
+public class DukeStorage {
+
+    /** The destination of the stored data */
+    public String path;
```

Comment:
Maybe you could set String path to private, and create a getter in this class. Would help with your encapsulation
```suggestion
    private String path;
```